Formula/g/gleam.rb

```diff
@@ -4,6 +4,7 @@ class Gleam < Formula
   url "https://github.com/gleam-lang/gleam/archive/refs/tags/v1.5.0.tar.gz"
   sha256 "0342babfbd6d8201ae00b6b0ef5e0b181bce5690c703ffae8dd02542e024c4c2"
   license "Apache-2.0"
+  revision 1
```

Comment:
What's this about? (for my education for potential future pull requests)

Reply:
It means that pre-existing `gleam` installs should now be considered outdated, which avoids their getting broken when `erlang` gets upgraded to Erlang 27 (because pre-existing `gleam` installs are still configured to use the `erlang` formula and not `erlang@26`).

Reply:
Got it.